app/observers/aaib_report_observers_registry.rb

```diff
@@ -1,5 +1,7 @@
-require "email_alert_exporter"
 require "formatters/aaib_report_publication_alert_formatter"
+require "formatters/aaib_report_indexable_formatter"
+require "markdown_attachment_processor"
```

Comment:
Since these are in them all, can we move this and the next line to the `AbstractSpecialistDocumentObserversRegistry`?